modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Chart.js

```diff
@@ -120,10 +120,10 @@ function legendFormatterGenerator(
 						backgroundColor: keyToHexColor(value),
 					}}
 				></span>
-				<span className="mr-2 text-secondary">
+				<span className="text-secondary">
 					{keyToTranslatedLabelValue(value)}
 				</span>
-				<span className="font-weight-bold">
+				<span className="font-weight-bold inline-item-after">
```

Comment:
Just to make sure that this change was expected in this PR 🙂.

Reply:
Yes :)